data/catalogs/deltares_data.yml

```
@@ -1531,249 +1661,268 @@ simard_v2011:
+      bd_sl6: 0.001
+      bd_sl7: 0.001
+      oc_sl1: 0.1
+      oc_sl2: 0.1
+      oc_sl3: 0.1
+      oc_sl4: 0.1
+      oc_sl5: 0.1
+      oc_sl6: 0.1
+      oc_sl7: 0.1
+      ph_sl1: 0.1
+      ph_sl2: 0.1
+      ph_sl3: 0.1
+      ph_sl4: 0.1
+      ph_sl5: 0.1
+      ph_sl6: 0.1
+      ph_sl7: 0.1
+  - version: 2.0
+    path: soil/soilgrids_v2.0/*/*_mean.vrt
+    meta:
+      paper_doi: https://doi.org/10.5194/soil-2020-65
+      paper_ref: de Sousa et al. (2020)
+      source_license: CC BY 4.0
+      source_url: https://www.isric.org/explore/soilgrids/faq-soilgrids
+      processing_script: soil/soilgrids_v2.0/src/preprocess_soilgrids_v2020.ipynb
+      processing_notes: soil/soilgrids_v2.0/src/readme.txt
+    rename:
+      bdod_0-5cm_mean: bd_sl1
+      bdod_5-15cm_mean: bd_sl2
+      bdod_15-30cm_mean: bd_sl3
+      bdod_30-60cm_mean: bd_sl4
+      bdod_60-100cm_mean: bd_sl5
+      bdod_100-200cm_mean: bd_sl6
+      clay_0-5cm_mean: clyppt_sl1
+      clay_5-15cm_mean: clyppt_sl2
+      clay_15-30cm_mean: clyppt_sl3
+      clay_30-60cm_mean: clyppt_sl4
+      clay_60-100cm_mean: clyppt_sl5
+      clay_100-200cm_mean: clyppt_sl6
+      soc_0-5cm_mean: oc_sl1
+      soc_5-15cm_mean: oc_sl2
+      soc_15-30cm_mean: oc_sl3
+      soc_30-60cm_mean: oc_sl4
+      soc_60-100cm_mean: oc_sl5
+      soc_100-200cm_mean: oc_sl6
+      phh2o_0-5cm_mean: ph_sl1
+      phh2o_5-15cm_mean: ph_sl2
+      phh2o_15-30cm_mean: ph_sl3
+      phh2o_30-60cm_mean: ph_sl4
+      phh2o_60-100cm_mean: ph_sl5
+      phh2o_100-200cm_mean: ph_sl6
+      silt_0-5cm_mean: sltppt_sl1
+      silt_5-15cm_mean: sltppt_sl2
+      silt_15-30cm_mean: sltppt_sl3
+      silt_30-60cm_mean: sltppt_sl4
+      silt_60-100cm_mean: sltppt_sl5
+      silt_100-200cm_mean: sltppt_sl6
+      sand_0-5cm_mean: sndppt_sl1
+      sand_5-15cm_mean: sndppt_sl2
+      sand_15-30cm_mean: sndppt_sl3
+      sand_30-60cm_mean: sndppt_sl4
+      sand_60-100cm_mean: sndppt_sl5
+      sand_100-200cm_mean: sndppt_sl6
+      SoilThickness_250_mean: soilthickness
+      TAXOUSDA_250_mean: tax_usda
+    unit_mult:
+      bd_sl1: 0.01
+      bd_sl2: 0.01
+      bd_sl3: 0.01
+      bd_sl4: 0.01
+      bd_sl5: 0.01
+      bd_sl6: 0.01
+      bd_sl7: 0.01
+      oc_sl1: 0.01
+      oc_sl2: 0.01
+      oc_sl3: 0.01
+      oc_sl4: 0.01
+      oc_sl5: 0.01
+      oc_sl6: 0.01
+      oc_sl7: 0.01
+      ph_sl1: 0.1
+      ph_sl2: 0.1
+      ph_sl3: 0.1
+      ph_sl4: 0.1
+      ph_sl5: 0.1
+      ph_sl6: 0.1
+      ph_sl7: 0.1
+      clyppt_sl1: 0.1
+      clyppt_sl2: 0.1
+      clyppt_sl3: 0.1
+      clyppt_sl4: 0.1
+      clyppt_sl5: 0.1
+      clyppt_sl6: 0.1
+      clyppt_sl7: 0.1
+      sltppt_sl1: 0.1
+      sltppt_sl2: 0.1
+      sltppt_sl3: 0.1
+      sltppt_sl4: 0.1
+      sltppt_sl5: 0.1
+      sltppt_sl6: 0.1
+      sltppt_sl7: 0.1
+      sndppt_sl1: 0.1
+      sndppt_sl2: 0.1
+      sndppt_sl3: 0.1
+      sndppt_sl4: 0.1
+      sndppt_sl5: 0.1
+      sndppt_sl6: 0.1
+      sndppt_sl7: 0.1
 
 vito:
```

```suggestion
vito_2015:
```
To match the rest better?

---

this is not the version but the epoch